rpcs3/Emu/Cell/lv2/sys_rsx.h

```diff
@@ -84,11 +84,9 @@ struct RsxDmaControl
 	be_t<u32> unk1;
 };
 
-struct alignas(16) RsxSemaphore
+struct RsxSemaphore
 {
-	be_t<u32> val;
-	be_t<u32> zero;
-	be_t<u64> zero2;
+	atomic_be_t<u32> val;
```

Comment:
Why not keep it 16 bytes long?

Reply:
It represents a single NV406E semaphore. gcm restricts it to 16-byte because of NV4097 methods restrictions but even there the underlying size is 4 bytes, so its just an alignment restriction.